test/binder/binder_test.cpp

```diff
@@ -36,7 +36,7 @@ using std::make_tuple;
 namespace peloton {
 namespace test {
 
-class BinderCorrectnessTest : public PelotonTest {
+class BinderCorrectnessTestTests : public PelotonTest {
```

Comment:
This class should've been names `BinderCorrectnessTests`!

Reply:
@schedutron Can you apply this change and rebase the PR? How many changes of the test files are still missing?

It's time that we merge it in and finish that part.

Reply:
Sorry for getting this delayed. I'll apply the changes soon.